src/utils/SortingAlgorithms.js

```diff
@@ -75,6 +75,8 @@ export function quickSort(items){
      }
      
      quickSortHelper(resultItems, 0, resultItems.length - 1);
+     
+     results.push(results[results.length-1]);
```

Comment:
This is added to the result to the result array to bring it back to the original color state at the end of all swappings in the UI.